compiler/ballerina-parser/src/test/resources/expressions/query-expr/query_expr_assert_122.json

```diff
@@ -411,44 +408,42 @@
               ]
             },
             {
-              "kind": "INVALID_EXPRESSION_STATEMENT",
-              "hasDiagnostics": true,
-              "diagnostics": [
-                "ERROR_INVALID_EXPRESSION_STATEMENT"
-              ],
+              "kind": "LOCAL_VAR_DECL",
```

Comment:
Lets add a test where the semicolon before this line is not present int the `.bal` file